backend/controllers/postController.js

```diff
@@ -88,6 +88,58 @@ deleteTag = async function(req,res){
 }
 
 
+addMilestone = async function(req,res){
+    try{
+	milestone = {
+	    project_id : req.body.projectId,
+	    date : req.body.date,
+	    title : req.body.title,
+	    description : req.body.description,
+	    type : req.body.type
+	}
+	milestoneDb = await ProjectMilestone.create(milestone)
+	res.status(201).send({message : "Milestone is added"})
+	}catch(error){
+	    res.status(500).send({"error": error})
+	}
+}
+
+
+updateMilestone = async function(req,res){
+    var fieldsToUpdate = {};
+    for(var field of Object.keys(req.body)){
```

Comment:
if you are not doing any modifications, fieldsToUpdate seems redundant. also instead of using for of. you could use shorter methods such as object.assign, or even simpler, just an equals sign type of assignment.

Reply:
Changed it, thank you for recommendation.